libs/utils/env.py

```diff
@@ -600,6 +601,14 @@ def _init_target_gem5(self):
             system['platform']['description']))[0]
         self.conf['board'] = self.conf['board'].lower() + suffix
 
+        energy_file = self.conf['board'] + '_energy';
+        try:
+            self.nrg_model = globals()[energy_file]
```

Comment:
1. It is a good idea to have it default to existing "real" platform and/or energy file, but it won't apply for Juno, unfortunately (please see 2). 
2. I think this is possible for it to use the juno.json file instead of the gem5juno.json file, but not the juno "real" energy file. If you look over the gem5juno_energy.py file you'll see that the order of cores is different and the frequencies are slightly different from the ones of the real platform. This is due to how GEM5 accounts for time (which more knowledgeable people can explain better).
The reason I would choose not to reuse the juno.json file is that by precisely specifying cores and  big_core in gem5juno.json I make sure that this works for older versions of gem5 that had an issue with differentiating cores.
But I'm not sure it's worth the duplication in this case. I'll submit another version with 1.

Thank you!